schema_dev.yml

```diff
@@ -1,9 +1,8 @@
 ruby:
-    - 2.1.8
+    - 2.3.1
 activerecord:
-    - 4.2.0
-    - 4.2.1
-    - 4.2.6
+    - 4.2
+    - 5.0
```

Comment:
Why not keep testing 4.2.0, 4.2.1, and 4.2.6?  (so that the coverage is at 100%)